src/core/PassphraseGenerator.cpp

```diff
@@ -98,7 +109,23 @@ QString PassphraseGenerator::generatePassphrase() const
     QStringList words;
     for (int i = 0; i < m_wordCount; ++i) {
         int wordIndex = randomGen()->randomUInt(static_cast<quint32>(m_wordlist.length()));
-        words.append(m_wordlist.at(wordIndex));
+        tmpWord = m_wordlist.at(wordIndex);
+
+        //convert case
+        switch (m_wordCase) {
+        case 0 : // lower case
```

Comment:
Prefer not to use "magic numbers" instead define an enum in the class:
```
enum PassphraseCase
{
	LOWERCASE,
	UPPERCASE,
	TITLECASE
};
```

Reply:
changes made